radix-engine/src/kernel/substate_io.rs

```diff
@@ -18,8 +18,9 @@ use radix_substate_store_interface::db_key_mapper::SubstateKeyContent;
 use sbor::prelude::Vec;
 use sbor::rust::collections::BTreeSet;
 use sbor::rust::collections::LinkedList;
+use sbor::Sbor;
 
-#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Debug, Copy, Clone, Sbor, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
I think that this would be addressed by the `DebugTransactionReceipt` idea I mentioned [here](https://github.com/radixdlt/radixdlt-scrypto/pull/1834#issuecomment-2178658074). If we do it then we would no longer need to make those models SBOR encodable.